usr/share/rear/restore/COMMVAULT/default/400_restore_with_galaxy.sh

```diff
@@ -0,0 +1,91 @@
+#
+# restore data with Commvault
+#
+
+# verify that we have a backupset
+[ "$COMMVAULT_BACKUPSET" ]
+StopIfError "Commvault  Backup Set not defined [COMMVAULT_BACKUPSET=]."
+
```

Comment:
Better don't use ...IfError functions in new code.
See my comment in lib/_input-output-functions.sh
https://github.com/rear/rear/blob/rear-2.7/usr/share/rear/lib/_input-output-functions.sh#L794

Reply:
OK, but if we don't actually look at `$?`, is it still not OK to use it?

Reply:
See my comment in lib/_input-output-functions.sh
https://github.com/rear/rear/blob/rear-2.7/usr/share/rear/lib/_input-output-functions.sh#L794
that reads (excerpt):
```
# The ...IfError functions fail when 'set -e' is set
# cf. https://github.com/rear/rear/issues/534
# for example code like
#   set -e
#   COMMAND
#   StopIfError "COMMAND failed"
# cannot work because 'set -e' exits the script when COMMAND results non-zero exit code
# so that the subsequent StopIfError is never reached.
```